scripts/test.sh

```diff
@@ -6,6 +6,7 @@ set -ex -o pipefail
 (
   cd integration/project
   yarn
+  yarn ngcc
```

Comment:
Hm.. I guess we need this, but this will slow down CI a bit.

Reply:
yeah, it's gonna take more time for CI to run, but as we add more tests for Ivy, this will become necessary.